src/main/java/jp/co/yahoo/yosegi/block/PushdownSupportedBlockWriter.java

```diff
@@ -85,6 +87,12 @@ public void setup( final int blockSize , final Configuration config ) throws IOE
       defaultConfig.compressorClass =
           FindCompressor.get( config.get( "spread.column.maker.default.compress.class" ) );
     }
+    if ( config.containsKey( "compress.optimize.allowed.ratio" ) ) {
+      double allowedRatio = config.getDouble( "compress.optimize.allowed.ratio" , 1.25d );
```

Comment:
I think that it is better not to specify this value directly but to be able to set it by the external setting file.
But, if this parameter realizes good performance, it seems good to set initial setting.